qa/tasks/cephfs/fuse_mount.py

```diff
@@ -435,17 +433,16 @@ def find_socket(client_name):
             client_name="client.{0}".format(self.client_id))
 
         # Find the admin socket
-        p = self.client_remote.run(args=[
-            'sudo', 'python3', '-c', pyscript
-        ], stdout=StringIO(), timeout=(15*60))
-        asok_path = p.stdout.getvalue().strip()
+        asok_path = self.client_remote.sh([
+            'sudo python3 -c', pyscript
```

Comment:
Which syntax? I guess it is just incorrect usage of even inhuman syntax, this code has two disadvantages: 1) using python3 on remote host, we better use shell snippets to be python independent ; 2) Don't use remote.run([]), use remote.sh("") ; 3) for big python snippets it is better to prior copy them to a temp file and call it instead of using 'python3 -c'.

Reply:
thanks for the pointing this out

Reply:
> Which syntax?
```py
self.client_remote.run(args=['sudo', 'python3', '-c', pyscript], ...)
```

@kshtsk i think we should limit the scope of the change instead of trying to humanize the inhuman tests, unless it helps to make the python3 port easier. in this case, how about leaving embedded python3 script intact and use something like
```py
```py
self.client_remote.run(args="sudo python3 -c '{}'".format(pyscript), ...)
```

Reply:
@tchaikov yeah, I guess it was a "typo" from my side during merge :-) reverted this line to the original